invenio_oauthclient/contrib/keycloak/handlers.py

```diff
@@ -45,7 +45,7 @@
 from .helpers import get_user_info
 
 
-def info_serializer_handler(remote, resp, user_info, **kwargs):
+def info_serializer_handler(remote, resp, token_user_info, user_info, **kwargs):
```

Comment:
breaking change: the order does not change since previously the user info was from the token, however, the name of the paramter changes.